index.html

```diff
@@ -321,6 +321,88 @@ <h2>Securing JSON-LD VCs with COSE</h2>
       </section>
     </section>
 
+
+
+<section class="normative">
+  <h2>Key Discovery</h2>
+  <p>
+In order to complete the <a data-cite="VC-DATA-MODEL#dfn-verify">verification</a> process, 
+a <a data-cite="VC-DATA-MODEL#dfn-verifier">verifier</a> needs to obtain the cryptographic keys used to secure the 
+<a data-cite="VC-DATA-MODEL#dfn-credential">credential</a>.
+  </p>
+  <p>
+There are several different ways to discover the verification keys of
+the <a data-cite="VC-DATA-MODEL#dfn-issuers">issuers</a>
+and <a data-cite="VC-DATA-MODEL#dfn-holders">holders</a>.
```

Comment:
These lines state that there are ways to discover the verification keys. Yet, after reading the next sections, I wouldn't be able to describe the specifics of a way to do discovery. Am I missing something, or is that the intention?

Should readers be expected to be able to answer the question: "how can one go from `vc-jwt` to `verification key`"?

Reply:
i think the idea is to add that in a follow on PR

Reply:
@andresuribe87 you are not missing anything... I am trying to work our way towards that.... By first documenting how registered claim names are used, and second document, the various (and there are many) ways they are used to discovery keys....

Concretely, in future PRs (needed to close the many open related issues), I hope to cover:

0. `kid` (by itself)
1. `iss` (by itself)
2. `iss` and `kid`
3. did urls
4. well known endpoints
5. relative and absolute urls

It seems natural to start with explaining how these terms are used in general, before providing more detailed examples.

To get an idea of the kind of things we need to say here,  take a look at:

https://w3c.github.io/vc-data-integrity/#retrieve-verification-method

Except our case is worse, since we cannot assume JSON-LD processing of the dereferenced resources.

We will aim to document compatible with JSON-LD resources AND the other mechanisms for discovering keys that are not related to JSON-LD or DIDs.

Goal is to document what people do, and then try to make a recommendation to avoid increasing the implementation burden for the spec.

Reply:
Excellent! That makes sense to me. Ship it